### PR TITLE
11.1 Release Blog

### PR DESCRIPTION
**Pull request by michaelstaib**
- Adds the ChilliCream Platform 11.1 release blog.
- Updates the change-log.

### CODE REVIEW

**Comment by arif-hanif** [comment]
@michaelstaib looks good

File: website/src/blog/2021-03-31-hotchocolate-11-1/2021-03-31-hotchocolate-11-1.md [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+When using the new type in our annotation-based approach, you only need to use the new interface `IFile`.
+
+```csharp
+public record CreateNewUserInput(string Username, IFile ProfilePicture);
+
+public class Mutation
+{
+    public async Task<NewUserPayload> CreateNewUser(CreateNewUserInput input)
+    {
+        using Stream stream = input.OpenReadStream();
```
Comment (tobias-tengler) [inline website/src/blog/2021-03-31-hotchocolate-11-1/2021-03-31-hotchocolate-11-1.md:150]:
```suggestion
        using Stream stream = input.ProfilePicture.OpenReadStream();
```

Comment (michaelstaib) [reply website/src/blog/2021-03-31-hotchocolate-11-1/2021-03-31-hotchocolate-11-1.md]: this is now fixed.

**Comment by michaelstaib** [comment]
@tobias-tengler do you have a twitter handle?

**Comment by tobias-tengler** [comment]
> @tobias-tengler do you have a twitter handle?

Yes [tobiastengler](https://twitter.com/tobiastengler), but I'm not really active :D

**Pull request merged**